gbpservice/nfp/configurator/lib/schema.py

```diff
@@ -34,14 +34,6 @@
 """
 
 
-skip_kwargs_validation_for = [
-                                'firewall',
```

Comment:
This was useful when service_type was used to tell if the request was for
generic config or not. Since we did away with that, we no longer require
this. Now, in schema_validator, we check if *not is_generic_config, then
skip*.

Regards,
Deepak

On Thu, Apr 28, 2016 at 4:58 PM, njagadish <notifications@github.com> wrote:

> In gbpservice/nfp/configurator/lib/schema.py
> <https://github.com/oneconvergence/group-based-policy/pull/263#discussion_r61411559>
> :
>
> > @@ -34,14 +34,6 @@
> >  """
> >
> >
> > -skip_kwargs_validation_for = [
> > -                                'firewall',
>
> Is this not needed? we are using this for avoiding validation for some
> cases right?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/oneconvergence/group-based-policy/pull/263/files/1c747a3c87fe61766d3df4f577ec4a2b55b958c3#r61411559>
>